tests/test_codex_feed_hooks.py

```
@@ -420,6 +420,138 @@ def test_codex_monitor_exits_when_workspace_has_no_surfaces(cli_path: str, root:
+    session_id = f"codex-monitor-invalid-owner-pid-session-{os.getpid()}"
+    env = os.environ.copy()
+    env["CMUX_SOCKET_PATH"] = str(socket_path)
+    env["CMUX_WORKSPACE_ID"] = FAKE_WORKSPACE_ID
+    env["CMUX_CODEX_MONITOR_UNRESOLVED_TRANSCRIPT_TIMEOUT_SECONDS"] = "30"
+
+    with FakeCmuxSocket(
+        socket_path,
+        None,
+        surfaces=[{"id": FAKE_SURFACE_ID, "ref": FAKE_SURFACE_ID}],
+    ) as fake:
+        result = subprocess.run(
```

Ruff (0.15.12)

[error] `subprocess` call: check for execution of untrusted input

(S603)